### PR TITLE
added generic wildflyswarm application

### PR DESCRIPTION
**Pull request by sunilk747**

### CODE REVIEW

**Comment by sunilk747** [comment]
@animuk Could you please review it.

**Comment by gastaldi** [comment]
I added the `do not merge` label until the @fabric8-launcher/runtime-wildfly-swarm team reviews this PR

**Comment by kenfinnigan** [comment]
What's the purpose of this Booster?

And any reason why it wasn't discussed with the team responsible for the runtime first?

**Comment by sunilk747** [comment]
@kenfinnigan This is a generic wildly swarm application.This application contains only the core dependencies required for a wildly swarm application. On top of it user can add any dependencies and create a application of his choice. @animukh can add more to this.

**Comment by animuk** [comment]
@kenfinnigan  We are planning to introduce a blank booster concept where user can start their application from the scratch. In this booster, We only have core dependencies for wildfly swarm listed in the pom.

**Comment by gastaldi** [comment]
@animuk @sunilk747 sorry, but I have some questions about this PR.

Boosters provided by RedHat are usually supported by the respective runtime team (who is going to be responsible for maintaining this booster in case of changes/new releases?). Second, OSIO has a requirement of using productized boosters. This booster specifically is using the community version, is that intended? Also this booster belongs to a new mission, here are the steps to follow when creating new boosters: https://github.com/fabric8-launcher/launcher-documentation/wiki/Creating-a-new-Booster

In short, I believe there should be some discussion before adding a new mission as we don't want to end up with unmaintainable code. 

Coincidence or not, the Launcher team is also looking for this same mission as part of our next releases, so to avoid extra work and future headaches, I'd appreciate if you could kick-start with a thread in the fabric8 launcher mailing list explaining what this Booster needs to do, etc. 

Let's keep working as a team, as we always do ;)

**Comment by Ladicek** [comment]
I believe the `rest-http` boosters look exactly like what you're after here, and your code is very close to the existing `rest-http` boosters.

**Comment by lholmquist** [comment]
if a booster is suppose to show off a piece of functionally, what does a blank booster provide then?

**Comment by kenfinnigan** [comment]
@gastaldi raises important points around maintenance, version of the implementation, etc

I'd also like to raise concerns about it being in a repo outside the runtime team's ability to easily remedy if there's a problem, and also that it's using `master` and not a tag.

**Comment by maxandersen** [comment]
@kenfinnigan can you point @sunilk747 on where it is best he makes a PR to add the blank booster ?

@sunilk747 having this inside fabric8-analytics does not make much if any sense. analytics is not related to this at all.
